Server/service/jobQueue.js

```
@@ -139,9 +146,7 @@ class JobQueue {
       return { jobs, load };
     } catch (error) {
       error.service === undefined ? (error.service = SERVICE_NAME) : null;
-      error.method === undefined
-        ? (error.method = "getWorkerStats")
-        : null;
+      error.method === undefined ? (error.method = "getWorkerStats") : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)